docs/FAQ.md

```diff
@@ -6,8 +6,11 @@ p = quilt3.Package()
 p.set_dir(".", ".")
 p.push("USR/PKG", message="MSG", registry="s3://BUCKET")
 ```
-> Use a [.quiltignore file](https://docs.quiltdata.com/advanced-usage/.quiltignore)
+
+{% hint style='tip' %}
+Use a [.quiltignore file](https://docs.quiltdata.com/advanced-usage/.quiltignore)
```

Comment:
Can you not use a relative link? We generally don't abs link like this to our own docs.